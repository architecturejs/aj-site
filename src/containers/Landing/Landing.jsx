import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import './Landing.scss';

export default class Landing extends Component {

  constructor() {
    super();
  }

  render () {
    return (
      <div className='Landing d-flex flex-column align-items-center justify-content-center'>
        <h1>Architecture JS</h1>
      </div>
    );
  }

}