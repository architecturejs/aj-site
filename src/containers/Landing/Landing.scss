.Landing {
  height: -webkit-calc(100vh - 32px);
  height:    -moz-calc(100vh - 32px);
  height:         calc(100vh - 32px);
}
