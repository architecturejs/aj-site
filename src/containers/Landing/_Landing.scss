header.masthead {

  height: 100vh;
  position: relative;
  width: 100%;
  min-height: auto;
  text-align: center;
  color: white;
  background-image: url("../../assets/img/architecturejs.jpg");
  background-position: center;
  filter: grayscale(1);

  @include background-cover;

  .header-content {
    position: relative;
    width: 100%;
    padding: 150px 15px 100px;
    text-align: center;
    .header-content-inner {
      h1 {
        font-size: 30px;
        font-weight: 700;
        margin-top: 0;
        margin-bottom: 0;
        text-transform: uppercase;
      }
      hr {
        margin: 30px auto;
      }
      p {
        font-size: 16px;
        font-weight: 300;
        margin-bottom: 50px;
        color: fade-out(white, .3);
      }
    }
  }

  @media (min-width: 768px) {

    .header-content {
      position: absolute;
      top: 50%;
      padding: 0 50px;
      -webkit-transform: translateY(-50%);
      -ms-transform: translateY(-50%);
      transform: translateY(-50%);
      .header-content-inner {
        max-width: 1000px;
        margin-right: auto;
        margin-left: auto;
        h1 {
          font-size: 50px;
        }
        p {
          font-size: 18px;
          max-width: 80%;
          margin-right: auto;
          margin-left: auto;
        }
      }
    }
  }
}
/*.Landing {
  background: url("../../assets/img/achitecturejs.jpg");
  height: -webkit-calc(100vh - 32px);
  height:    -moz-calc(100vh - 32px);
  height:         calc(100vh - 32px);
}*/
