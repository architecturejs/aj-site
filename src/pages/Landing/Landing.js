import React, { Component } from 'react';

import './Landing.scss';

class Landing extends Component {
  render () {
    return (
      <section className="h-screen flex flex-col justify-center">
        <h1 className="text-center text-5xl">Architecture JS</h1>
      </section>
    );
  }
}

export default Landing;
