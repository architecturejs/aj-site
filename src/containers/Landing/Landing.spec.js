import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import Landing from './Landing';

test('Landing snapshot test', () => {

  const component = shallow(<Landing />);
  const tree = shallowToJson(component);

  expect(tree).toMatchSnapshot();

});