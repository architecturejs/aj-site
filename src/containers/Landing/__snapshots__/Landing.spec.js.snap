// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapshot test 1`] = `
<header
  className="masthead"
>
  <div
    className="header-content"
  >
    <div
      className="header-content-inner"
    >
      <h1
        className="text-shadow"
        id="homeHeading"
      >
        Architecture JS
      </h1>
    </div>
  </div>
</header>
`;
